fauna/assets/logs/fauna_tests.yaml

```diff
@@ -0,0 +1,34 @@
+id: "fauna"
```

Comment:
The ident for this file is wrong (result property should be ident more).
You can copy this to fix it. you will receive some updates from the CI afterwards.

```
id: "fauna"
tests:
 -
  sample: |-
    {
      "ddsource":"fauna",
      "service":"fauna",
      "message": {
        "API_VERSION": "10",
        "REGION_GROUP": "us-std",
        "DATABASE": ["JsDriverTests", "StreamTestDB"],
        "REQUEST_HEADERS": "{\"x_forwarded_for\":\"38.124.75.155, 10.1.1.45\",\"user_agent\":\"node-fetch/1.0 (+https://github.com/bitinn/node-fetch)\"}",
        "REQUEST_METHOD": "POST",
        "RESPONSE_CODE": "200",
        "TS": "2024-06-24T17:19:52.790106Z",
        "TAGS":{},
        "FQL_SHAPE": "Collection.all()",
        "QUERY_TIME_MS":15,s
        "BYTES_IN":30,
        "BYTES_OUT":453,
        "BYTE_READ_OPS":10,
        "BYTE_WRITE_OPS":0,
        "COMPUTE_OPS":1,
        "TXN_RETRIES":0,
        "TRACEPARENT": "00-69db13d3da034748ffda8ab23f3726ca-7785c136a489a882-00",
        "RATE_LIMITS_HIT":[]
      }
    }
  result: null
```